README.rst

```diff
@@ -66,3 +66,18 @@ Using pip_:
 .. _pip: http://www.pip-installer.org/en/latest/
 .. _here: http://azkabancli.readthedocs.org/
 .. _examples: https://github.com/mtth/azkaban/tree/master/examples
+
+Development
+------------
+
+Run tests:
+
+  nosetests
```

Comment:
This won't render as code, let's enclose it in a code block:

```
.. code-block:: bash

  $ nosetests
```

Same just below for the config file snippet.

Reply:
Thanks, didn't realize it wasn't `.md`